L1Trigger/CSCTriggerPrimitives/src/CSCComparatorDigiFitter.cc

```diff
@@ -0,0 +1,290 @@
+#include "L1Trigger/CSCTriggerPrimitives/src/CSCComparatorDigiFitter.h"
+#include "L1Trigger/CSCCommonTrigger/interface/CSCConstants.h"
+#include "DataFormats/Math/interface/normalizedPhi.h"
+
+namespace
+{
+  // CSC LCT patterns
+  // the number quotes the distance to the center
+  // 999 is invalid
+
+  std::vector<std::vector<int> > pat0delta {  
```

Comment:
hi @dildick - it seems that none of these 999 are needed. The vectors are only used to find if the other values are present or not(?)